tests/core/shader.test.js

```diff
@@ -0,0 +1,261 @@
+/* global AFRAME */
+/* global assert, process, setup, suite, test, teardown */
+var entityFactory = require('../helpers').entityFactory;
+var componentName = 'shader';
+
+var VIDEO = 'https://ucarecdn.com/bcece0a8-86ce-460e-856b-40dac4875f15/'; // 'base/tests/assets/test.mp4';
+
+function isEmpty (map) {
```

Comment:
You could just do `!Object.keys(map).length` without needing a separate fn that's only called twice